app/decorators/suscribir/suscripcion_decorator.rb

```diff
@@ -0,0 +1,13 @@
+module Suscribir
+  class SuscripcionDecorator < Draper::Decorator
+    delegate_all
+
+    def url_desuscribir
+      h.engine_suscribir.pedir_confirmacion_baja_url(id, token)
```

Comment:
:+1: for me. Although I found this interesting info on this topic: http://nepalonrails.com/blog/2013/11/using-rails-route-helpers-inside-engine